src/components/projectSelect/ProjectOptionGroup.tsx

```diff
@@ -37,9 +39,19 @@ export const ProjectOptionGroup: React.FC<OptionGroupProps> = ({
   name,
   projects,
   onClick,
+  repoIdentifier,
+  isProjectSettingsPage = false,
 }) => (
   <OptionGroupContainer>
-    <Overline>{name}</Overline>
+    {/* if it's the project settings page and it's not the "" group, make the header clickable */}
+    {repoIdentifier === "" || !isProjectSettingsPage ? (
```

Comment:
This logic can be replaced by a simple conditional prop.

Reply:
how so?

Reply:
I think I know what you mean, I committed the change that I think you meant.